bin/service-data/specification/get-schemas.js

```diff
@@ -0,0 +1,17 @@
+const path = require('path')
+
+module.exports = function getSchemas (specsPath = process.cwd()) {
+  const schemas = []
+
+  try {
+    const {
+      specifications
+    } = require(path.join(specsPath, 'package.json'))
+    specifications.path = specsPath
+    schemas.push(specifications)
+  } catch (e) {
+    // no package.json
```

Comment:
is it worthwhile logging a warning, or is that not useful?

Reply:
I've wondered about that. Ordinarily, I would always log, but AR doesn't. It's probably an idea to start changing that

Reply:
Done. New rule: delete AR's commented code, add logging in his exceptions

Reply:
(I try not to fiddle with his code too much or else I'll never stop, but I had fiddled in one of those chunks and I still didn't fix the logging, doh)